### DISCUSSION TITLE
Live db flag

### DISCUSSION THREAD

**Discussion by deanzod** [Sequel-Ace/Sequel-Ace#1214] [Ideas]
It would be really useful to be able to mark a database as "live/production" to lock it down a bit e.g. throw a warning popup if you try and run an import or delete a table, row etc... Sometimes I have come really close to wiping a production database thinking it is the local version.

**Comment by jensjns** [comment]
I use the color-tagging options to mark DB connections like this:

Red = Production
Orange/yellow = Staging or similar
Green/untagged = Local/throw away

This makes it really obvious in the sidebar menu what you're connecting to. The color tag shows on the connection tab if you're connected to multiple databases but there's no indication if you're only connected to a single database, which would be handy.

With that said I think a flag for production DB's is a good idea. What actions is prevented should also be chosen by the user I think.

**Reply by deanzod** [reply]
Yeah I do exactly that currently but when I have local and live open in two tabs (or even just live open in its own window) the colour coding is useless.

I'm thinking something in connection setup like 'mark as production/local' then in settings there can be a few checkboxes for production or even both e.g. alert confirm on import, on delete row, on update etc...

**Reply by dnzprmksz** [reply]
I am using the same color coding protocol. But I think this is a good idea. It would be great if it asks for confirmation before modifications on production database.

**Reply by Kaspik** [reply]
You can always have color visible even with single tab - "View" - "Show tab bar"
